### DISCUSSION TITLE
能够正常登陆，并且已成为管理员，但是评论提交时出现语法错误

### DISCUSSION THREAD

**Discussion by dongla-danzeng** [walinejs/waline#1506] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/72214248/196020195-2ed0c442-d7e0-4fbb-a970-f4dc5c703351.png)

**Comment by lizheming** [comment, marked as answer]
如何部署的，提供地址看一下，这是你的接口报错了。

**Reply by dongla-danzeng** [reply]
已解决，问题出在代码的缩进上，谢谢！
